src/main/webapp/web.xml

```diff
@@ -3,6 +3,9 @@
          xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
                       http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd"
          version="4.0">
+    <session-config>
+        <session-timeout>5</session-timeout>
+    </session-config>
```

Comment:
did you need to add this configuration according to the task's description?

Reply:
just for test